sdk/servicebus/azure-servicebus/azure/servicebus/aio/_base_handler_async.py

```diff
@@ -385,3 +386,9 @@ async def close(self) -> None:
         """
         await self._close_handler()
         self._shutdown.set()
+        try:
+            self._service_bus_client._handlers.remove(self)  # pylint: disable=protected-access
```

Comment:
if using `set()` when iterating through handlers, does this mean that multiple/duplicates of the same handler can be added to the list? In that case, would you want to iterate through the list to remove all instances of `self` since `remove` only removes the first element/`self`?

(Same comment for sync)

Reply:
duplicates of the same handler CAN NOT be add to the list. sorry for the confusion, the set is used by me for dynamic deleting, I'll remove that 😉

Reply:
same handlers could not be added, see my comment above.